README.md

```diff
@@ -69,7 +69,7 @@ Options:
 
 `markdown-inject` expands a given glob for markdown files. Then it discovers the below `CODEBLOCK` HTML comments within each markdown file, performs the appropriate action (in this case, reading another local file), and writes content back into the markdown file:
 
-<!-- CODEBLOCK_START_USAGE {"ignore": true} -->
+{/* CODEBLOCK_START_USAGE {"ignore": true} */}
```

Comment:
this MDX comment start is... matching the HTML comment end on line 79?

this makes the README actually render the text instead of hiding it

<img width="793" alt="image" src="https://github.com/target/markdown-inject/assets/298435/4027ca42-c868-4f99-b107-1004d65c825a">